tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -3809,6 +3843,36 @@ def test_nvfp4_4gpus_online_eplb(self, moe_backend, mtp_nextn):
 
+@pytest.mark.timeout(14400)
+@pytest.mark.skip_less_device(8)
+@pytest.mark.skip_less_device_memory(140000)
+@skip_pre_blackwell
+class TestDeepSeekV4Pro(LlmapiAccuracyTestHarness):
+    MODEL_NAME = "deepseek-ai/DeepSeek-V4-Pro"
+    MODEL_PATH = f"{llm_models_root()}/DeepSeek-V4-Pro"
+    EXTRA_EVALUATOR_KWARGS = dict(
+        apply_chat_template=True,
+        system_prompt=_DEEPSEEK_V4_GSM8K_SYSTEM_PROMPT,
+    )
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)